packages/lit-html/src/directive-helpers.ts

```diff
@@ -59,6 +64,16 @@ export const isDirectiveResult = (
     ? (value as DirectiveResult)?._$litDirective$ !== undefined
     : (value as DirectiveResult)?._$litDirective$ === klass;
 
+/**
+ * Tests is a part is a single-expression part.
```

Comment:
```suggestion
 * Tests whether a part has a single-expression and no strings to interpolate between.
```

Reply:
This is tricky; this function returns true if there's one value AND empty strings to either side of it. So there's two conditions to meet.

Reply:
done